user/tests/wiring/no_fixture_spi/spi.cpp

```diff
@@ -391,3 +391,117 @@ test(SPI_10_SPI1_Begin_With_Slave_Ss_Pin)
 }
 #endif // Wiring_SPI1
 
+// Performance tests. All SPI interfaces share the same driver. Testing on SPI only is sufficient.
+test(SPI_11_SPI_Transfer_10000_Bytes_No_Locking_Less_Than_135_Ms)
+{
+    SPI.setClockSpeed(5, MHZ);
+    SPI.begin();
+    system_tick_t start = millis();
+    for(unsigned int i=0; i < 10000; i++)
+    {
+        SPI.transfer(0x55);
+    }
+    Serial.printf("%d ms\r\n", millis() - start);
+    assertTrue(millis() - start < 135);
```

Comment:
Let's not use magic numbers in all the tests and instead calculate this number based on the clock speed + % of overhead.